src/components/Task/Drawer/CommentList/CommentList.tsx

```diff
@@ -76,8 +82,8 @@ const TaskDrawerCommentList = ({
                 </CardContent>
               </Card>
             )}
-            {data.task.comments.nodes.length > 0 &&
-              data.task.comments.nodes.reduce((result, comment) => {
+            {data?.task.comments.nodes?.length > 0 &&
+              data.task.comments.nodes?.reduce((result, comment) => {
```

Comment:
This is defined if the last condition was true
```suggestion
              data.task.comments.nodes.reduce((result, comment) => {
```

Reply:
Yeah I might have been overly cautious here, but you're right.